### PR TITLE
link user to comment

### PR DESCRIPTION
**Pull request by panayotovyura**

### CODE REVIEW

**Comment by Ishugaev** [comment]
approved

File: app/Resources/views/blog/post_show.html.twig [hunk 1]
```diff
@@ -47,7 +47,7 @@
     {% for comment in post.comments %}
         <div class="row post-comment">
             <h4 class="col-sm-3">
-                <strong>{{ comment.authorDisplayName ? comment.authorDisplayName : comment.authorEmail }}</strong> {{ 'post.commented_on'|trans }}
+                <strong>{{ comment.user.displayName ? comment.user.displayName : comment.user.email }}</strong> {{ 'post.commented_on'|trans }}
```
Comment (holyspecter) [inline app/Resources/views/blog/post_show.html.twig]: i'd propose to put this in `__toString()` method of User entity, so here we may just say `{{ comment.user }}`

Comment (panayotovyura) [inline app/Resources/views/blog/post_show.html.twig]: I want to hold this in my heands, so I can use user name or email here.

File: src/AppBundle/Controller/BlogController.php [hunk 2]
```diff
@@ -134,10 +134,15 @@ public function commentNewAction(Request $request, Post $post)
         $form->handleRequest($request);
 
         if ($form->isSubmitted() && $form->isValid()) {
+
+            // todo: we need to have user from database in session
```
Comment (holyspecter) [inline src/AppBundle/Controller/BlogController.php]: user should already be there. Is `$this->getUser()` here returning `null`?

File: src/AppBundle/Entity/User.php [hunk 3]
```diff
@@ -155,6 +170,25 @@ public function isAdmin()
     }
 
     /**
+     * @return mixed
```
Comment (holyspecter) [inline src/AppBundle/Entity/User.php]: string

File: src/AppBundle/Controller/BlogController.php (hunk 2 above)
Comment (panayotovyura) [inline src/AppBundle/Controller/BlogController.php]: no, but this user is not user from database((( I created issue for that

Comment (holyspecter) [inline src/AppBundle/Controller/BlogController.php]: hmm... that's weird :(

**Pull request merged**